lib/rewrite/rewrite-expr.c

```diff
@@ -48,15 +53,12 @@ log_rewrite_queue(LogPipe *s, LogMessage *msg, const LogPathOptions *path_option
     {
       self->process(self, &msg, path_options);
     }
-  if (G_UNLIKELY(debug_flag))
-    {
-      value = log_msg_get_value(msg, self->value_handle, &length);
-      msg_debug("Rewrite expression evaluation result",
-                evt_tag_str("value", log_msg_get_value_name(self->value_handle, NULL)),
-                evt_tag_printf("new_value", "%.*s", (gint) length, value),
-                evt_tag_str("rule", self->name),
-                log_pipe_location_tag(s));
-    }
+  value = log_msg_get_value(msg, self->value_handle, &length);
```

Comment:
Please add the if (debug_flag) around this, because this lookup is on a heavy path.

Reply:
The "value" not needed anymore, deleted it